README.md

```diff
@@ -81,6 +81,8 @@ The function supports the following keyword arguments:
 
 - ``tol``: tolerance of changes upon convergence (default = ``1.0e-6``).
 
+- ``replicates``: Number of times to repeat factorization (default = ``1``).
```

Comment:
This wording makes it sound like the factorization is repeated 1 time, i.e. performed 2 times. I'd suggest this instead:
```suggestion
- ``replicates``: Number of times to perform factorization (default = ``1``).
  Further replications are not performed once the algorithm has converged.
```